istio/src/main/scala/io/buoyant/k8s/istio/IstioConfigurator.scala

```diff
@@ -0,0 +1,52 @@
+package io.buoyant.k8s.istio
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.Stack.Params
+import com.twitter.finagle.buoyant.H2
+import com.twitter.finagle.param.Label
+import com.twitter.finagle.{Address, Name}
+import com.twitter.logging.Logger
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.ClientConfig
+import io.buoyant.k8s.istio.mixer.MixerClient
+
+trait IstioConfigurator {
```

Comment:
As far as I can tell, nothing there depends on the current object's state.  Could these methods just be static method instead?  (ie turn this trait into an object)

I tend to shy away from the mixin pattern when possible.

Reply:
Hm.. that's a good point. I think this had some template pattern logic at first but was gradually removed. Let me explore it, brb

Reply:
Done!